src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,171 @@
+package duke;
+
+import java.util.Scanner;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+
+/**
+ * Duke is a personal chatbot to keep track of things.
+ *
+ * @author Aaron Tan
+ */
+public class Duke extends Application {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+    private Parser parser;
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/User.jpg"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/Duke.jpg"));
+
+    /**
+     * Constructs an instance of Duke.
+     */
+    public Duke() {
+        ui = new Ui();
+        storage = new Storage();
+        parser = new Parser();
+    }
+
+    @Override
+    public void start(Stage stage) {
```

Comment:
I do believe the code can be made even better if you were to extract out the UI code and put that in the Ui.java file! But that would just be my personal preference!